specifyweb/specify/urls.py

```diff
@@ -13,6 +13,9 @@
     url(r'^specify/merge/status/(?P<merge_id>[0-9a-fA-F-]+)/$', views.merging_status),
     url(r'^specify/merge/abort/(?P<merge_id>[0-9a-fA-F-]+)/$', views.abort_merge_task),
 
+    #check for institution at login
+    url(r'^specify/institution/$', views.get_institution),
+
```

Comment:
Creating a new route `api/specify/institution` or `api/specify/institutions` might not be a good idea since we already have a route `api/specify/institution` which returns institution data for an authorized user. Especially the route name without the s could potentially override the existing route and create side effects

How about a route called `is_new_user` instead which directly checks the number of institutions and returns a boolean?